module-data/src/main/java/org/lindeck/data/model/cards/CardExtended.java

```diff
@@ -0,0 +1,18 @@
+package org.lindeck.data.model.cards;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.Entity;
+
+/**
+ * @author derovi
+ * This card has answer, that shows hust by click.
+ */
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Data
+public class CardExtended extends AbstractCard {
+    private String answer;
+}
```

Comment:
sooooo
i think "CARD WITH ANSWER" is no exists,  it is better to create a "ROTATED CARD". (we need to discuss it )